packages/venia-concept/src/ServiceWorker/registerMessageHandlers.js

```diff
@@ -0,0 +1,5 @@
+import { registerImagePreFetchHandler } from './Utilities/imageCacheHandler';
```

Comment:
You should be able delete everything after this line.

```suggestion
export { registerImagePreFetchHandler as default } from './Utilities/imageCacheHandler';
```